homeassistant/components/rtsp_to_webrtc/__init__.py

```diff
@@ -18,20 +18,32 @@
 
 from __future__ import annotations
 
+import asyncio
+import contextlib
 import logging
 from typing import Any
 
+from aiortc import (
```

Comment:
Do you think `aiortc` is a feasible long-term solution or should we explore a [plug-in](https://developers.home-assistant.io/docs/supervisor#architecture) for HA OS (like we have for audio)

Reply:
I think it can be, but too soon to tell yet.  Once we get the `ha-av` and `av` fork issue sorted out, we can likely move part of this forward and get it out into the wild (not as a default) so we can get feedback on it.  I expect this one is going to be a long exploration.  Ideally it does work out to use `aiortc` so that it can be feasible for container (and core) users as well.